### ISSUE TITLE
Error Message after update pytorch v1.11.0

### ISSUE THREAD

**Issue by zmce2018** [timeseriesAI/tsai#565]
Hi Team, 

I have found the following error message. 
```
fcst = TSForecaster(X, y, splits=splits, batch_tfms=batch_tfms, bs=128, arch=TSTPlus, loss_func = mse, metrics=[mse, mae,smape],arch_config={'dropout':0.3,"fc_dropout":0.9,'n_layers':2,"d_ff":128},device='cuda', cbs=ShowGraph()) 

  File E:\ANA\envs\tsai\lib\site-packages\tsai\tslearner.py:176 in __init__
    model = build_ts_model(arch, dls=dls, device=device, verbose=verbose, pretrained=pretrained, weights_path=weights_path,

  File E:\ANA\envs\tsai\lib\site-packages\tsai\models\utils.py:143 in build_ts_model
    c_in = ifnone(c_in, dls.vars)

  File E:\ANA\envs\tsai\lib\site-packages\fastcore\basics.py:459 in __getattr__
    if attr is not None: return getattr(attr,k)

  File E:\ANA\envs\tsai\lib\site-packages\fastcore\basics.py:459 in __getattr__
    if attr is not None: return getattr(attr,k)

  File E:\ANA\envs\tsai\lib\site-packages\fastai\data\core.py:461 in __getattr__
    def __getattr__(self,k): return gather_attrs(self, k, 'tls')

  File E:\ANA\envs\tsai\lib\site-packages\fastcore\transform.py:165 in gather_attrs
    if not res: raise AttributeError(k)
```

**Comment by oguiza** [comment]
Hi @zmce2018,
I'm afraid I can't help you out with the information you have provided. 
It's strange since I've recently upgraded tsai to work with Pytorch 1.12.
What version of tsai are you using? Could you share a code snippet to reproduce your error?

**Comment by zmce2018** [comment]
@oguiza thank you for your time. The problem is solved by updating PyTorch to 1.12. I am now closing this issue. 

thanks again.
